scripts/Diffraction/isis_powder/routines/common.py

```diff
@@ -596,6 +596,7 @@ def _load_raw_files(run_number_string, instrument, file_ext=None):
     :return: A list of loaded workspaces
     """
     run_number_list = generate_run_numbers(run_number_string=run_number_string)
+    file_ext = "" if file_ext is None else file_ext
```

Comment:
Could the `file_ext` parameter could simply be assigned as `""` here where it is declared in the parameters?

Reply:
It could. The `None` check is still required though as the caller of this picks it up from setting where it is either a `str` or `None` so it wouldn't really help much. The other places I changed to `""` are called from here so I was in control of what they receive. Could we leave it like this?